scripts/bundlesize/bundlesize.config.json

```diff
@@ -2,7 +2,7 @@
     "files": [
         {
             "path": "packages/lwc/dist/engine-dom/umd/es2017/engine-dom.min.js",
-            "maxSize": "16.62KB",
+            "maxSize": "16.72KB",
```

Comment:
what's the criteria to set this number?

Reply:
This limit is here to make sure the LWC engine bundled size doesn't grow inadvertently (eg. adding a new dependency by mistake). FYI @nolanlawson.

Reply:
Right, IIRC @jmsjtu and I just picked a number that was a bit above our current size. It's just supposed to be a warning so we don't inadvertently blow up the size.

Reply:
@nolanlawson from what I remember, last time we just picked it based on x% higher than the current size.